calcom/cal.com
# Mutually exclusive options

## What does this PR do?

Recurring events and Stripe payment should not be enabled at the same time, as there is no workflow put in place to make sense on how would that work, charging by series, by booking, rescheduling, cancelling a few occurrences, etc.

This PR makes sure each option on event type is mutually exclusive.

## Type of change




## How should this be tested?

 - Go to a Payment is enabled event type configuration and see recurring event option is gone. 
 - When Payment option is disabled, recurring event option should be in place.
